### DISCUSSION TITLE
The response was redacted to meet Responsible AI policies.

### DISCUSSION THREAD

**Discussion by BLTC-520** [community/community#64160] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Latest version of VS code, latest version of Github Copilot Chat. 

My question: 
Can you code a tic-tac-toe game in C++?

Github Copilot replies: 
The response was redacted to meet Responsible AI policies. If this was unexpected, please give feedback. 

Wondering is my prompt got problem, or the extension itself has some limits? I threw the same question into GPT-3.5, there is no problem for generating the code.

**Comment by BLTC-520** [comment]
Minor bug. The copilot itself back to normal in 3 minutes.

**Comment by RiverExplorer** [comment]
Still down for me ...

**Reply by improwise** [reply]
Same here, broken.

**Comment by maqdk** [comment]
Same here as well. Broken

**Reply by BLTC-520** [reply]
Try refresh?
